sdk/servicebus/azure-messaging-servicebus/src/test/java/com/azure/messaging/servicebus/ServiceBusMessageSerializerTest.java

```diff
@@ -132,6 +139,108 @@ void deserializeMessage() {
         assertEquals(payload, actualMessage.getBody().toString());
     }
 
+    /**
+     * Verifies that we can serialize OffsetDateTime, Duration and URI in application properties.
+     */
+    @Test
+    void serializeMessageWithSpecificApplicationProperties() {
+        String contents = "some contents";
```

Comment:
also test with something with 2 or more byte in utf8

Reply:
I didn't fix the `sizeOf` method in this PR, so I just keep the same test content as former code.

Reply:
OK. URL does not allow non-ascii. So you are safe here.